src/components/View.jsx

```diff
@@ -204,13 +204,22 @@ const View = ({ data, id, path }) => {
     onChangeFormData,
   });
 
+  const formid = `form-${id}`;
+
   useEffect(() => {
     if (submitResults?.loaded) {
       setFormState({
         type: FORM_STATES.success,
         result: intl.formatMessage(messages.formSubmitted),
       });
       captcha.reset();
+      const formItem = document.getElementById(formid);
+      const formItemPosition = formItem.getBoundingClientRect();
+      window.scrollTo({
+        top: formItemPosition.x,
+        left: formItemPosition.y,
+        behavior: 'smooth',
+      });
```

Comment:
`document.getElementById` can return `null`, theoretically. This will not happen when using the default implementation, but guarding this code against that is very easy to achieve, so I would suggest doing that.
We never know what integrators can do with volto customizations :)